deploy/base/lookup-artifact-location-v0.1.yaml

```diff
@@ -17,12 +17,12 @@ spec:
       type: string
       description: >-
         The image to use for dependency analysis.
-      default: "quay.io/sdouglas/hacbs-jvm-build-request-processor:dev"
+      default: "quay.io/dwalluck/hacbs-jvm-build-request-processor:dev"
```

Comment:
This should not have been necessary, the value here is replaced by the dev-template generated overlay.

Reply:
Done.